pkg/golinters/govet.go

```diff
@@ -143,7 +143,8 @@ func NewGovet(settings *config.GovetSettings) *goanalysis.Linter {
 	return goanalysis.NewLinter(
 		"govet",
 		"Vet examines Go source code and reports suspicious constructs, "+
-			"such as Printf calls whose arguments do not align with the format string",
+			"such as Printf calls whose arguments do not align with the format string. "+
+			"This is the same analysis as the 'go vet' tool",
```

Comment:
```suggestion
			"It uses the same passes as 'go vet'.",
```

The sentence inside my [comment](https://github.com/golangci/golangci-lint/discussions/4363#discussioncomment-8413385) was a popularization for you but it's not something that we should write in a doc because it's not the reality.

Golancgi-lint is based on the internal tooling of `go vet`, and it also uses the passes.
"Analysis" is a specific term inside the technical context of golangci-lint and linters, which is not adapted to this situation.

Reply:
Thanks, I used a variant of your suggestion below.